portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,37 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addGreeting() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
```

Comment:
Does this still work since you changed "/data" to "/comments" in DataServlet.java?

Reply:
Oops, that method is from a past tutorial and I'm not using it anymore. I'll delete it.